.circleci/config.yml

```
@@ -77,18 +84,22 @@ jobs:
   linux-python-39-signac-10:
     <<: *test-template
     environment:
+      DEPENDENCIES: "NEWEST"
       SIGNAC_VERSION: "signac~=1.0.0"
   linux-python-39-signac-13:
     <<: *test-template
     environment:
+      DEPENDENCIES: "NEWEST"
       SIGNAC_VERSION: "signac~=1.3.0"
   linux-python-39-signac-14:
     <<: *test-template
     environment:
+      DEPENDENCIES: "NEWEST"
       SIGNAC_VERSION: "signac~=1.4.0"
```

We aim to test the two most recent minor versions. Since we're updating this file anyway, we can change that now.
```suggestion
  linux-python-39-signac-15:
    <<: *test-template
    environment:
      DEPENDENCIES: "NEWEST"
      SIGNAC_VERSION: "signac~=1.5.0"
  linux-python-39-signac-14:
    <<: *test-template
    environment:
      DEPENDENCIES: "NEWEST"
      SIGNAC_VERSION: "signac~=1.4.0"
```